### DISCUSSION TITLE
increase the number of columns

### DISCUSSION THREAD

**Discussion by myracullu** [Eugeny/tabby#9381] [General]
Forgive me if this feature exists, but I would like to know where I can increase the number of columns. This adjustment allows the terminal to not crash on Huawei devices when accessed.

**Comment by geraldotdsjr** [comment]
Hello,

I would like talk about this configuration too. Columns ajustment is a nice resource.

Save from Brazil dudes.

**Comment by igorbf495** [comment]
Hello. I'm also interested in the ability to adjust the number of columns. I've noticed that this feature could help prevent stability issues on specific devices, such as mentioned for Huawei devices. If anyone knows how to make this adjustment, I would greatly appreciate it if you could share the instructions with us. Thank you.

**Comment by adicao** [comment]
It also happens to me. Is there any ETA for solution update?

**Comment by myracullu** [comment]
Hello something news about this ?

**Comment by Eugeny** [comment]
What do you guys even mean?

**Reply by myracullu** [reply]
https://github.com/Eugeny/tabby/discussions/9192 , this is error happened because the number of columns cant be ajusted , in windterm x putty , mobaxterm this dont happen. In windterm this is fix ajustament colums numbers of terminal ...

**Reply by myracullu** [reply]
1.0.222 and problem still ...
